docs/rst/user_manual/configuration.rst

```diff
@@ -0,0 +1,300 @@
+        type: local     # Participant Type = simple
+        domain: 3       # DomainId = 3
+
+    ################
+
+    simple:             # Participant Id = simple ; Participant Type = simple
+        domain: 6       # DomainId = 6
+
+The first Participant `Participant0` has Participant Id *Participant0* and Participant Type *simple*.
+The second Participant has Participant Id *simple*, and type is not required to be specified as it is get from the
+Participant Id.
+
+.. note::
+
+    Participant Type is not case sensitive.
+    A Participant called *Simple* would be of type ``simple``.
+
+.. note::
+
+    The Participant Id is get as Participant Type when type is not specified.
+    If type is explicitly specified, the Participant Id is not used to get the type.
+
+
+Network Address
+===============
+
+Network Addresses are elements that can be configured for specific Participants.
+An Address is set by:
+
+* *IP*: IP of the host (public IP in case of WAN).
+* *Port*: Port where the Participant is listening.
+* *Transport Protocol*: ``UDP`` or ``TCP``.
+  If it is not set, it would be chosen by default depending on the Participant Type.
+* *IP version*: ``v4`` or ``v6``.
+  If it is not set, it would be chosen depending on the *IP* string format.
+
+.. code-block:: yaml
+
+    ip: "127.0.0.1"
+    port: 11666
+    transport: "tcp"
+    ip-version: "v4"
+
+    ################
+
+    ip: "2001:4860:4860::8844"      # It is recognized as IPv6
+    port: 1616
+
+.. warning::
+
+    ``ip`` field does not allow DNS names, only well-formed IP addresses.
+
+
+Discovery Server GuidPrefix
+===========================
+
+A :term:`Discovery Server` requires a DDS :term:`GuidPrefix` in order to other Participants connect to it.
+There are several possibilities for configuring a GuidPrefix.
+
+
+Discovery Server GuidPrefix by string
+-------------------------------------
+
+Using tag ``guid``, the GuidPrefix of the Discovery Server will be set as it.
+Be aware of using the correct format for GuidPrefix.
+That is, 12 hexadecimal numbers (lower than ``ff``) separated with ``.``.
+
+.. code-block:: yaml
+
+    guid: "1.f.1.0.0.0.0.0.0.0.ca.fe"     # GuidPrefix = 01.0f.01.00.00.00.00.00.00.00.ca.fe
+
+
+Discovery Server GuidPrefix by Id
+---------------------------------
+
+Using tag ``id``, the GuidPrefix will be calculated arbitrarily using a default |ddsrouter| GuidPrefix.
+This default GuidPrefix is ``01.0f.x.00.00.00.00.00.00.00.ca.fe`` where ``x`` is the value of the id.
+Default value for ``id`` is ``0``.
+
+.. code-block:: yaml
+
+    id: 13                          # GuidPrefix = 01.0f.0d.00.00.00.00.00.00.00.ca.fe
+
+.. note::
+
+    In the actual version only 256 different ids are allowed.
+    In future releases it would be implemented to allow further ids.
+
+
+ROS Discovery Server GuidPrefix
+-------------------------------
+
+There is a specific GuidPrefix for ROS2 executions, so it could be used using Fast DDS CLI and
```

Comment:
In general, ROS 2 is written separated.
```suggestion
There is a specific GuidPrefix for ROS 2 executions, so it could be used using Fast DDS CLI and
```